### PR TITLE
rework front-end robot cards to display on hover

### PR DESCRIPTION
**Pull request by kej-jay**
## Changes
The robot cards now only display only the robots name. Upon hovering on the card the usual options (start and edit robot, edit robot name) are shown.

## Screenshots
before:
image

after:
https://user-images.githubusercontent.com/44369294/115224737-86042f00-a10d-11eb-8366-18e129b21142.mov

## Checklist before merge

**Developer's responsibilities**
* [ ] **Assign** one or two developers
* [ ] **Change code** if reviewer(s) has/have requested it
* [ ] **Pull request build** has passed
* [ ] **tested locally** (in at least chrome & firefox if frontend)
* [ ] updated the **documentation**
* [ ] added **tests** where necessary

### CODE REVIEW

**Review by lukashueller — APPROVED** [review]

File: client/src/components/content/RobotContainer/RobotContainer.jsx [hunk 1]
```diff
@@ -87,7 +87,7 @@ const RobotContainer = (props) => {
       {!hoveredUpon && (
         <Col className={[styles.box, styles.robotBox]}>
           <Row justify='center' align='middle' style={{ height: '100%' }}>
```
Comment (lukashueller) [inline client/src/components/content/RobotContainer/RobotContainer.jsx:89]: Please add this to align the Heading in the Box.
```suggestion
          <Row justify='center' align='middle' style={{ height: '100%', margin:'0.5rem' }}>
```

**Review by lukashueller — CHANGES_REQUESTED** [review]
Please fix the vertical alignment of the heading (see my suggestion) and also delete line 3 in the `module.css`-file. This line (`padding: 1rem;`) is no longer needed I think.

File: client/src/components/content/RobotContainer/RobotContainer.jsx (hunk 1 above)
Comment (WolfgangDaniel) [reply client/src/components/content/RobotContainer/RobotContainer.jsx:89]: +1

**Comment by WolfgangDaniel** [comment]
> 
> 
> Please fix the vertical alignment of the heading (see my suggestion) and also delete line 3 in the `module.css`-file. This line (`padding: 1rem;`) is no longer needed I think.

I would disagree with the padding part. If we delete the padding the boxes won't have space between them

**Review by WolfgangDaniel — APPROVED** [review]
Looks fine to me 👍

File: client/src/components/content/RobotContainer/RobotContainer.module.css [hunk 2]
```diff
@@ -1,6 +1,5 @@
 .box {
   text-align: center;
-  padding: 1rem;
   height: 13rem;
```
Comment (lukashueller) [inline client/src/components/content/RobotContainer/RobotContainer.module.css:3]:
```suggestion
  padding: 1rem;
  height: 13rem;
```

**Review by lukashueller — CHANGES_REQUESTED** [review]
Sorry, I didn't seem to have looked properly there. The line of code is needed after all, as I see in my local frontend.

**Review by lukashueller — APPROVED** [review]
Please rebase and then you can merge

**Review by WolfgangDaniel — APPROVED** [review]

**Pull request merged**